bin/epaa.py

```diff
@@ -1017,17 +1028,27 @@ def __main__():
         if version not in EpitopePredictorFactory.available_methods()[method]:
             raise ValueError("The specified version " + version + " for " + method + " is not supported by Fred2.")
 
+    thresholds = {"syfpeithi":50, "mhcflurry":500, "mhcnuggets-class-1":500, "mhcnuggets-class-2":500}
```

Comment:
I think we have to add `netMHC` here as well otherwise it will fail if these tools are used. 
`netmhc, netmhcpan, netmhcii, netmhciipan`